types/pangu/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for pangu 3.3
+// Project: https://github.com/vinta/pangu.js
+// Definitions by: York Yao <https://github.com/plantain-00>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function spacing(text: string): string;
+
+export function spacingFile(path: string, callback: (error: Error, data: string) => void): void;
+export function spacingFile(path: string): Promise<string>;
+
+export function spacingFileSync(path: string): string;
+
+export function spacingText(text: string, callback: (error: Error, data: string) => void): void;
+export function spacingText(text: string): Promise<string>;
+
+export function spacingPage(): void;
+export function spacingElementById(id: string): void;
+export function spacingElementByClassName(className: string): void;
+export function spacingElementByTagName(tagName: string): void;
+
+export namespace Pangu {
+}
+export default Pangu;
```

Comment:
First this package does not seem to have a `default` export at runtime

```sh
> var p = require("pangu");
undefined
> p;
NodePangu { filePrefix: 'readable.', Pangu: [Function: NodePangu] }
> p.default
undefined
```

Second, the `default` export here is an empty namespace, not sure what that adds.

Reply:
The `default` and `namespace` are just removed.
The document said it can be used like `import pangu from 'pangu';`(https://github.com/vinta/pangu.js#nodejs), so I thought it should have default export.